e2e/pages/adf/filePreviewPage.ts

```diff
@@ -89,16 +89,10 @@ export class FilePreviewPage {
 
         this.checkCanvasWidth().then((width) => {
             actualWidth = width;
```

Comment:
If the expect was removed, i don't think actualWidth is still useful. From my point of view, this zoomIn() method has no sense. I would instead do these steps in test. Add an expect in the test related to width and height in case it's necessary, then clickZoomIn(), then add other expects in test.

Reply:
the 'actualWidth' is still needed, but I do understand your point that the 'expect' should better be inside the 'it(...)' from the actual test file...